build-system/compile/single-pass.js

```diff
@@ -580,7 +579,9 @@ exports.singlePassCompile = async function(entryModule, options) {
     })
     .then(compile)
     .then(wrapMainBinaries)
-    .then(postProcessConcat)
+    .then(intermediateBundleConcat)
+    .then(eliminateIntermediateBundles)
+    .then(thirdPartyConcat)
```

Comment:
The phased concatting is necessary because third_party code can use an `_` object/parameter/namespace, etc. So, it's not safe for me to do the pruning with that code already injected.